source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,348 @@
+.. _realm-sync-migrate-modes:
+
+=========================
+Migrate Device Sync Modes
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 3
+   :class: singlecol
+
+Atlas App Services Apps can have one of two Device Sync modes: Partition-Based Sync
+and Flexible Sync. Partition-Based Sync is an older mode and you should consider
+migrating to Flexible Sync.
+
+Migrating an App that uses Partition-Based Sync to Flexible Sync is an automatic
+process. Other than upgrading the SDK version, migrating Sync modes doesn't require 
+any changes to your client app code.
+
+Migration results in Partition-Based Sync clients communicating with a
+Flexible Sync backend. We recommend eventually :ref:`migrating your client app 
+<realm-sync-migrate-client>` to also use Flexible Sync instead of
+Partition-Based Sync.
+
+Requirements
+------------
+
+App Services App:
+
+- Currently use Partition-Based Sync
+
+Linked Atlas Cluster:
+
+- Use MongoDB 5.0 or later for Flexible Sync compatibility
+- Have storage auto-scaling enabled
+- Be an M10+ Cluster Tier
+- Have time-based oplog setting enabled
+
+Client apps:
+
+- Use ``realm-core`` vTBD+, which means these SDK versions: TBD
+
+.. _realm-sync-migrate-what-to-know:
+
+What to Know Before Migrating
+-----------------------------
+
+Migrating your App's Sync mode involves changes that affect both client apps and
+your App's backend. You should be aware of the side effects and plan ahead.
+
+- Client apps that use an SDK with with a realm-core version lower than TBD will
+  not be able to sync after migration.
+- How long a migration takes is directly proportional to the amount of data that
+  needs to be migrated. The more data, the longer it will take to migrate.
+- Migrations can be :ref:`canceled <realm-sync-migrate-cancel>` during the 
+  migration or :ref:`reverted <realm-sync-migrate-revert>` within a certain 
+  timeframe after the migration is finished.
+- Progress notifications will not work after migration because Flexible Sync does
+  not support them.  
+- Client apps that were connected to the Partition-Based Sync backend will 
+  experience a client reset after the backend is migrated to Flexible Sync. You 
+  should use a :ref:`client reset <handle-a-client-reset>` handler with recovery enabled.
+- Creating new tables in a migrated Flexible Sync backend using 
+  :ref:`Development Mode <development-mode>` from a Partition-Based Sync client
+  may lead to unexpected behavior.
+
+.. _realm-sync-migrate-stages:
+
+Migration Stages
+~~~~~~~~~~~~~~~~
+
+There are three stages to a Sync mode migration. You can see your migration's
```

Comment:
Consider switching the order of these first two sentences. That way, the second one is a lead-in to the list that follows. Might even consider ending it with a colon (:)

Reply:
Excellent suggestion. Thank you!